packages/langium-vscode/esbuild.js

```diff
@@ -0,0 +1,24 @@
+//@ts-check
+const watch = process.argv.includes('--watch');
+const minify = process.argv.includes('--minify');
+const success = watch ? 'Watch build succeeded' : 'Build succeeded';
+
+require('esbuild').build({
+    // Two entry points, one for the extension, one for the language server
+    entryPoints: ['src/extension.ts', 'src/language-server/main.ts'],
+    outdir: 'out',
+    bundle: true,
+    loader: { '.ts': 'ts' },
+    external: ['vscode'], // the vscode-module is created on-the-fly and must be excluded.
+    platform: 'node', // VSCode extensions run in a node process
+    sourcemap: true,
+    watch: watch ? {
+        onRebuild(error) {
+            if (error) console.error('Watch build failed')
+            else console.log(success)
```

Comment:
Please add a timestamp so one can see when the last rebuild took place.

Reply:
Yep, good idea 👍